jjb/edgex-templates-docker.yaml

```diff
@@ -215,3 +217,5 @@
 
     triggers:
       - timed: 'H 11 * * *'
+      - reverse:
+          jobs: '{project-name}-{stream}-release-version-java-daily-no-sonar'
```

Comment:
Should remove the timed here since java-daily is already timed.

Reply:
Fixed.